bot/exts/events/advent_of_code/_cog.py

```diff
@@ -63,44 +69,28 @@ async def adventofcode_group(self, ctx: commands.Context) -> None:
     @adventofcode_group.command(
         name="subscribe",
         aliases=("sub", "notifications", "notify", "notifs"),
-        brief="Notifications for new days"
+        help=MOVED_HELP,
     )
     @whitelist_override(channels=AOC_WHITELIST)
     async def aoc_subscribe(self, ctx: commands.Context) -> None:
-        """Assign the role for notifications about new days being ready."""
-        current_year = datetime.now().year
-        if current_year != AocConfig.year:
-            await ctx.send(f"You can't subscribe to {current_year}'s Advent of Code announcements yet!")
-            return
-
-        role = ctx.guild.get_role(AocConfig.role_id)
-        unsubscribe_command = f"{ctx.prefix}{ctx.command.root_parent} unsubscribe"
+        """
+        Deprecated role command.
 
-        if role not in ctx.author.roles:
-            await ctx.author.add_roles(role)
-            await ctx.send(
-                "Okay! You have been __subscribed__ to notifications about new Advent of Code tasks. "
-                f"You can run `{unsubscribe_command}` to disable them again for you."
-            )
-        else:
-            await ctx.send(
-                "Hey, you already are receiving notifications about new Advent of Code tasks. "
-                f"If you don't want them any more, run `{unsubscribe_command}` instead."
-            )
+        This command has been moved to bot, and will be removed in the future.
+        """
+        raise MovedCommandError(MOVED_COMMAND)
 
     @in_month(Month.DECEMBER)
     @commands.guild_only()
-    @adventofcode_group.command(name="unsubscribe", aliases=("unsub",), brief="Notifications for new days")
+    @adventofcode_group.command(name="unsubscribe", aliases=("unsub",), help=MOVED_HELP)
     @whitelist_override(channels=AOC_WHITELIST)
     async def aoc_unsubscribe(self, ctx: commands.Context) -> None:
```

Comment:
Please just remove this command entirely and add it to the aliases of the other command. No need having two commands that do the same thing.

Reply:
https://github.com/python-discord/sir-lancebot/pull/972/commits/a309e6fbd497fe493ce6fca484c1d1ec20d2fb26